.github/workflows/build-go.yaml

```diff
@@ -0,0 +1,38 @@
+on:
+  push:
+    branches:
+      - main
+      - rc
+      - hotfix-rc
+
+  pull_request:
+
+jobs:
+  build:
+    name: Build
+    needs:
+      - build_rust
+      - generate-schemas
```

Comment:
⚠️  Dependencies should be updated, I don't see these being used in this workflow.

Reply:
Fixed.